package.json

```diff
@@ -37,7 +37,8 @@
   "license": "ISC",
   "dependencies": {
     "lodash.assign": "^4.2.0",
-    "stacktrace-js": "^1.1.0"
+    "stacktrace-js": "^1.3.1",
+    "react-native-source-maps": "git://github.com/philipshurpik/react-native-source-maps.git#master"
```

Comment:
I've done this in personal projects, but I'm hesitant to stick this into a published npm project (such as this one). Namely, because this project tends to not get updated frequently (I'm not actively using it anymore), and who knows what version-breaking stuff philip might be doing. :)

Is there a particular change you need from his project? Is it easy enough to ask him for a release, versus this change? He seems pretty active on github.

Reply:
The version on npm was not working for me. I don't remember well why but I think it was related to latest versions of reatc native (I'm using 0.49.3)

Reply:
@philipshurpik (can I tag you?) Any interest in releasing a newer version of your library to npm?

Reply:
Since I'm not sure if he gets notifications about tags on my repo, can you file an issue on `react-native-source-maps` asking for an npm release?